jgrapht-core/src/main/java/org/jgrapht/Graphs.java

```diff
@@ -42,7 +42,9 @@
 package org.jgrapht;
 
 import java.util.*;
+import java.util.function.Predicate;
 
+import org.jgrapht.experimental.dag.DirectedAcyclicGraph;
```

Comment:
We don't want dependencies from the main jgrapht packages on experimental.  So, the DAG-related methods should be moved to the DAG class itself (or a new companion utility class in the same package).

(Arguably, it's time to finally make DAG non-experimental, but we'll leave that for a separate effort.)